packages/perseus-editor/src/components/util.ts

```diff
@@ -58,6 +59,23 @@ export function getDefaultFigureForType(type: LockedFigureType): LockedFigure {
                 color: "blue",
                 filled: true,
             } as LockedPointType;
+        case "line":
+            return {
+                type: "line",
+                kind: "line",
+                points: [
+                    getDefaultFigureForType("point"),
+                    {
+                        ...getDefaultFigureForType("point"),
+                        coord: [2, 2],
+                    },
+                ],
+                color: "blue",
+                lineStyle: "solid",
+                showArrows: false,
+                showStartPoint: false,
+                showEndPoint: false,
+            } as LockedLineType;
```

Comment:
I don't think we need these `as ...` statements.  We're returning a type union and because each of these default objects has a `type: "..."` in them, Typescript should properly validate that they conform to the correct member of the union.